radio/apps/schedules/admin.py

```diff
@@ -78,9 +78,6 @@ def copy_ScheduleBoard(self, request, queryset):
     copy_ScheduleBoard.short_description = _("Make a Copy of calendar")
 
 
-admin.site.register(ScheduleBoard, ScheduleBoardAdmin)
-
-
```

Comment:
bunch admin registration at the end of module to not spread registration logic across the code